sdk/src/test/java/com/qonversion/android/sdk/QUserPropertiesManagerTest.kt

```diff
@@ -487,6 +527,22 @@ class QUserPropertiesManagerTest {
         }
     }
 
+    @Test
+    fun renewHandledProperties() {
+        // given
+        val handledProperties = mapOf("someKey7" to "someValue7")
+
+        every {
+            mockPropertiesStorage.getHandledProperties()
+        } returns handledProperties
+
+        // when
+        propertiesManager.renewHandledProperties()
```

Comment:
I think we should add something to handled properties before cleaning to test that it really clears something